.github/workflows/moodle-ci.yml

```diff
@@ -105,267 +118,114 @@ jobs:
-      - name: Initialise moodle-plugin-ci
-        run: |
-          composer create-project -n --no-dev --prefer-dist moodlehq/moodle-plugin-ci ci ^3
-          echo $(cd ci/bin; pwd) >> $GITHUB_PATH
-          echo $(cd ci/vendor/bin; pwd) >> $GITHUB_PATH
-          sudo locale-gen en_AU.UTF-8
-          echo "NVM_DIR=$HOME/.nvm" >> $GITHUB_ENV
-        env:
-          IGNORE_PATHS: classes/privacy,ignore,node_modules,integration,render
-          COVERAGE: false
-          CODECHECKER_IGNORE_PATHS: classes/privacy,ignore,node_modules,integration,render
-          PHPUNIT_IGNORE_PATHS: classes/privacy,ignore,node_modules,integration,render
-
-
-      # 3. Install Atto plugin ( and Quizzes ?)
-      - name: Add MathType plugin for Atto
-        id: install-plugin-atto
-        continue-on-error: true
-        run: |
-          moodle-plugin-ci add-plugin --branch ${GITHUB_REF##*/} wiris/moodle-atto_wiris
-      - name: Add MathType plugin for Atto using the stable branch
-        if: steps.install-plugin-atto.outcome != 'success'
-        run: |
-          moodle-plugin-ci add-plugin --branch stable wiris/moodle-atto_wiris
-
-      # 3.1 Install Tiny plugin ( and Quizzes ?)
-      - name: Add MathType plugin for TinyMCE
-        id: install-plugin-tiny
         continue-on-error: true
         run: |
-          moodle-plugin-ci add-plugin --branch ${GITHUB_REF##*/} wiris/moodle-tiny_wiris
-      - name: Add MathType plugin for TinyMCE using the stable branch
-        if: steps.install-plugin-tiny.outcome != 'success'
+          moodle-plugin-ci add-plugin --branch ${{ github.ref_name }} wiris/moodle-qtype_wq
+      - name: Add Wiris Quizzes plugin using the stable branch
+        if: steps.install-plugin-quizzes.outcome != 'success'
         run: |
-          moodle-plugin-ci add-plugin --branch stable wiris/moodle-tiny_wiris
+          moodle-plugin-ci add-plugin --branch main wiris/moodle-qtype_wq
 
       # 4. Install Moodle-plugin-ci
       - name: Install moodle-plugin-ci
-        run: |
-          moodle-plugin-ci install --plugin ./plugin --db-host=127.0.0.1
+        run: moodle-plugin-ci install --plugin ./plugin --db-host=127.0.0.1
         env:
           DB: ${{ matrix.database }}
           MOODLE_BRANCH: ${{ matrix.moodle-branch }}
 
-      # 5. Run all tests.
-      # 5.1. Lint
-      - name: PHP Lint
-        if: ${{ always() }}
-        run: moodle-plugin-ci phplint
-
-      # 5.2. PHP Detectors
-      - name: PHP Copy/Paste Detector
-        continue-on-error: true # This step will show errors but will not fail
-        if: ${{ always() }}
-        run: moodle-plugin-ci phpcpd
-
-      - name: PHP Mess Detector
-        continue-on-error: true # This step will show errors but will not fail
-        if: ${{ always() }}
-        run: moodle-plugin-ci phpmd
-
-      # 5.3. Code validations
-      - name: Moodle Code Checker
+      # 5.4. Code testing
+      # Run PHPUnit tests on all Moodle branches.
```

Comment:
Not very important, very at some point this numbered list became non-sense (for example, jumped from 4 to 5.4). Could you please check it out?

Reply:
True